src/SwitchField/SwitchField.js

```diff
@@ -105,5 +123,17 @@ FormSwitch.propTypes = {
         `Invalid prop ${propName} supplied to ${componentName}. Expected a string array of length 2.`
       );
     }
-  }
+  },
+  /**
+   * Custom styling for the MUI Switch component.
+   */
+  switchStyle: PropTypes.string,
```

Comment:
What's the purpose of this switchStyle prop? What would we use it for? If we need something non-standard then we just override the theme in the app where we need to. Also, should it not be an object anyway not a string? I think just remove this.